reference/marketing.v2.yml

```diff
@@ -42,7 +42,7 @@ paths:
       - Coupons
       summary: Get All Coupons
       description: |-
-        Returns a list of *Coupons*. Default sorting is by coupon/discount id, from lowest to highest. Optional filter parameters can be passed in.
+        Returns a list of *Coupons*. Default sorting is by coupon/discount id, from lowest to highest. You can pass in optional filter parameters. We recommended using `?min_id=x&limit=y` filters because the `?limit=x&page=x`filters can lead to performance issues.
```

Comment:
Maybe consider something along the line of 
```
We recommended using `?min_id=x&limit=y` to paginate through large set of data because it offers better performance.
```

Have a slightly positive spin 😄